SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -171,16 +174,18 @@ TArray<uint16> USpatialActorChannel::SkipOverChangelistArrays(FObjectReplicator&
 		}
 	}
 
-	return InitialRepChanged;
+	return { InitialRepChanged, *Replicator.RepLayout };
 }
 
-FPropertyChangeState USpatialActorChannel::CreateSubobjectChangeState(UActorComponent* Component)
+FHandoverChangeState USpatialActorChannel::CreateInitialHandoverChangeState(FClassInfo* ClassInfo)
```

Comment:
I think this function should be refactored to a separate helper function. There is no reason why this needs to be a member of the `SpatialActorChannel`. Right now it introduces a dependency on the `SpatialActorChannel` in the `SpatialSender` and the `SpatialGDKEditorGenerateSnapshot` for no reason.

Reply:
But I guess it is here for consistency as the `CreateInitialRepChangeState` has to be in the `SpatialActorChannel`